src/NuGet.Clients/NuGetConsole.Host.PowerShell/NuGetConsole.Host.PowerShell.csproj

```diff
@@ -179,12 +181,35 @@
       </PowerShellCmdletsHelpFiles>
     </ItemGroup>
   </Target>
+
+  <!-- This is called at PackVsix time to get localized files -->
   <Target Name="GetLocalizedPowerShellCmdletHelpFile">
     <ItemGroup>
+      <!-- Localized languages -->
+      <Lango Include="CHT" Locale="zh-Hant" />
+      <Lango Include="CHS" Locale="zh-Hans" />
+      <Lango Include="CSY" Locale="cs" />
+      <Lango Include="DEU" Locale="de" />
+      <Lango Include="ESN" Locale="es" />
+      <Lango Include="FRA" Locale="fr" />
+      <Lango Include="ITA" Locale="it" />
+      <Lango Include="JPN" Locale="ja" />
+      <Lango Include="KOR" Locale="ko" />
+      <Lango Include="PLK" Locale="pl" />
+      <Lango Include="PTB" Locale="pt-BR" />
+      <Lango Include="RUS" Locale="ru" />
+      <Lango Include="TRK" Locale="tr" />
```

Comment:
Is this an all-inclusive list of the languages we support? Did you copy it from somewhere? Maybe a comment explaining where you got this set.

Reply:
That's the list of VS supported languages we support. The list is defined at Microbuild.Plugins.Localization. Will add a comment.

Reply:
I tried to reference internally but, that would require to add imports:

```xml
  <Import Project="$(MicroBuildDirectory)MicroBuild.Core.props"/>
...
  <Import Project="$(MicroBuildDirectory)MicroBuild.Core.targets"/>
```

Reply:
I'm happy with just a comment somewhere :)